src/app/Dashboard/Dashboard.tsx

```diff
@@ -97,27 +155,27 @@ export function getConfigByTitle(title: String): CardConfig {
 
 export const Dashboard: React.FunctionComponent<DashboardProps> = (props) => {
   const dispatch = useDispatch<StateDispatch>();
-  const cardNames = useSelector((state: RootState) => state.dashboardConfigs.list);
+  const cardConfigs = useSelector((state: RootState) => state.dashboardConfigs.list);
```

Comment:
Are we planning to have these configurations specific for each target? I think right now its pretty much applying to all targets?

Reply:
Correct, it applies to all targets. That's by design. Perhaps this can be extended with multiple dashboard layouts, but the layouts would be identified by name rather than being associated with specific targets.

Reply:
Right that makes sense! Thanks for exlaining :D